src/freedv_api.c

```diff
@@ -649,7 +613,7 @@ int freedv_comprx(struct freedv *f, short speech_out[], COMP demod_in[]) {
     }
 
     if (FDV_MODE_ACTIVE( FREEDV_MODE_700D, f->mode)) {
-        rx_status = freedv_comp_short_rx_700d(f, (void*)demod_in, 0, 1.0);
+        rx_status = freedv_comp_short_rx_700d(f, (void*)demod_in, 0, 2.0f); // was 1.0 ??
```

Comment:
Why did you change this?

Reply:
I noticed everywhere else it was set to 2.0 (about 2 other places if I recall), so I thought it should be the same.

Reply:
Guess it's just a gain factor, which doesn't matter much for QPSK ...